src/lib/internal/helpers/store/effect.ts

```diff
@@ -10,18 +10,25 @@ import { safeOnDestroy } from '../lifecycle.js';
  * @template S - The type of the stores object
  * @param stores - The stores object to derive from
  * @param fn - The function to run when the stores change
+ * @param skipFirstRun - Whether to skip the first run
  * @returns A function that can be used to unsubscribe the effect
  */
 export function effect<S extends Stores>(
 	stores: S,
-	fn: (values: StoresValues<S>) => (() => void) | void
+	fn: (values: StoresValues<S>) => (() => void) | void,
+	skipFirstRun?: boolean
```

Comment:
Let's make this an object. @TGlide and I have spoke in the past about some other options we want to add to effects and by making it an object now it'll ease the addition of those.

```ts
type EffectOptions = {
  skipFirstRun?: boolean;
}
```